src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -227,50 +242,56 @@ public static AbstractBuilding create(Colony colony, TileEntityColonyBuilding pa
             {
                 BlockPos loc = parent.getPosition();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'loc' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)